server/backend/backend/settings.py

```
@@ -96,16 +97,29 @@
-        'PORT': config('DB_PORT', cast=int),
+if os.getenv('GITHUB_WORKFLOW') or os.getenv('CI'):
+    # If running inside GitHub Actions
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': os.getenv('POSTGRES_DB', 'test_db'),
+            'USER': os.getenv('POSTGRES_USER', 'postgres'),
+            'PASSWORD': os.getenv('POSTGRES_PASSWORD', 'postgres'),
+            'HOST': 'localhost',
+            'PORT': '5432',
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)